### ISSUE TITLE
Card button_attributes `aria-hidden="true"` is not being displayed 

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa#7742]
# Problem
The card button should have the `aria-hidden="true"` attribute but is only displaying `aria-hidden`.

# Steps to reproduce
Inspect markup of any card button. 

# Proposed solution
Look at the diff for `docroot/themes/custom/uids_base/templates/uids/card.html.twig`  in https://github.com/uiowa/uiowa/pull/6054/files and investigate why https://github.com/uiowa/uiowa/blob/main/docroot/modules/custom/uiowa_core/uiowa_core.module#L419 is not working.

**Comment by bspeare** [comment]
Resolved via https://github.com/uiowa/uiowa/pull/7782.
